tests/e2e/helpers_e2e_test.go

```diff
@@ -71,7 +71,7 @@ func calculateUncollectedFees(positionLiquidity, feeGrowthBelow, feeGrowthAbove,
 	// F_u = position_liquidity * (fee_growth_inside_t2 - fee_growth_inside_t1).
 	feesUncollected := positionLiquidity.Mul(feeGrowthInside.Sub(feeGrowthInsideLast))
 
-	return feesUncollected.TruncateDec()
+	return feesUncollected
```

Comment:
fees should be tracked as Dec. There was a test case where we tracked the uncollected fees from test A to B then from B to C. Uncollected fees from A to B was XXX.6 and uncollected fees from B to C was XXX.6. Because we only truncate after collection, we were expecting one unit less than what we actually got because the helper was truncating in between calls.